packages/storybook/stories/MessageThread/MessageThread.stories.tsx

```diff
@@ -339,13 +383,17 @@ const MessageThreadStory = (args): JSX.Element => {
       />
       {/* We need to use these two buttons to render more messages in the chat thread and showcase the "new message" button.
         Using storybook controls would trigger the whole story to do a fresh re-render, not just components inside the story. */}
```

Comment:
Should we update this comment after this change?

Reply:
Agree, in fact I think it can probably just be removed

Reply:
I updated the comment with explaining why we use FluentUI controls for sending messages instead of Storybook controls. I believe this can be really helpful for learning.